### ISSUE TITLE
Worlds Custom-tab Sorting Settings Broken/Unresponsive, stuck on Random

### ISSUE THREAD

**Issue by GearBell** [Yellow-Dog-Man/Resonite-Issues#2747]
### Describe the bug?

I noticed My Worlds tab became randomized a few weeks ago instead of showing in the chronological order they were made, and it continued to randomize every day. When selecting the Custom Search tab and scrolling down to Sorting Options, none of the options are clickable, it is stuck on Random.

### To Reproduce

Only a few people have noticed this issue so either it happens to some of us or its happening to all of us and only some of us have noticed. Select your My Worlds tab and go to Custom Search and scroll down to select any of the Sorting options such as Publish Date, etc - the bug will be that none of them are selectable.

### Expected behavior

The arrangement options should be responsive and change the arrangement of how the My Worlds are displayed.

### Resonite Version Number

2024.8.5.1341

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Vive

### Log Files

[DESKTOP-SH9HHOE - 2024.8.5.1341 - 2024-08-09 18_07_15.log](https://github.com/user-attachments/files/16568027/DESKTOP-SH9HHOE.-.2024.8.5.1341.-.2024-08-09.18_07_15.log)

### Additional Context

I did not use mods for this test, this bug applies to both Desktop and VR modes.

### Reporters

GearBell

**Comment by GearBell** [comment]
Just commenting to keep this alive, it is becoming an issue for work.

**Comment by GearBell** [comment]
Poking again to keep this alive

**Comment by Frooxius** [comment]
You don't need to poke it, it doesn't really help. Best thing is to ask people to vote on it. 

However if you can tell us a bit more on how it's an issue for work, that helps - if we get more context on how it's affecting people, this can help bump its priority.

**Comment by shiftyscales** [comment]
@Frooxius I suspect this is probably the same issue as #2512 and #2536, but it doesn't appear to affect freshly generated UIs, e.g. while not logged in. So I suspect it is a transient issue that happened during an upgrade at some point in the past- and I'm not sure if it'd be worth development time trying to chase it down when a dash reset / regenerating those particular facets will fix it.

My recommendation would be that you either reset your dash, or create / save new copies of those facets to replace the broken ones on your dash, @GearBell.

**Comment by GearBell** [comment]
Shifty makes an excellent point, I will try a dashboard reset. Apologies about the poke, I am not well versed of github and Ive seen people "poke" discord threads to keep them alive so I thought that was the norm. I will try a dashboard reset. How it is affecting my job is that I commonly make a few worlds a week and lose track of them and then cannot remember thier names and have to sift through a few hundred - with the option of it in order of creation date it helped me a lot. What was the command for dashboard reset, please?

**Comment by shiftyscales** [comment]
@GearBell - `-ResetDash` https://wiki.resonite.com/index.php?title=Command_Line_Arguments

**Comment by GearBell** [comment]
Thank you! This has solved the issue! I know there were 2 others having this issue I will pass this along. Thank you both for your patience! Im not sure how to close this ticket but consider it fixed!

**Comment by ko-tengu** [comment]
I think this may have been closed premptively since I managed to trigger this bug on an un-signed in account with an empty cache/data folder. After joining one world on the session the world browser stopped refreshing or allowing me to search sessions.

[BUFU - 2024.9.6.626 - 2024-09-06 13_59_53.log](https://github.com/user-attachments/files/16913258/BUFU.-.2024.9.6.626.-.2024-09-06.13_59_53.log)

**Comment by shiftyscales** [comment]
Did that persist between restarts, @ko-tengu? Do you know what timestamp / line the issue occurred in the above log? When submitting replication logs, it is helpful to press F3 before and after replication to make a marker in the log that makes it easier to find relevant parts of the log.

This issue also wasn't about searching sessions, but changing the filter mode on the bottom of the filter facet, e.g. random, by number visits, by published date, etc.

Either way- I agree this issue can stay open for the moment- I consider resetting the dash / manually regenerating the facets to be at best a workaround, not a solution for now.

**Comment by TisFoolish** [comment]
Is there a way to regenerate the specific facet without doing a -ResetDash?

**Comment by shiftyscales** [comment]
Yeah, you can apply the component to an empty slot and then save the corresponding facet to your inventory, @TisFoolish.

**Comment by ko-tengu** [comment]
It was at some point after 2:01:24 PM.306 ( 77 FPS), ln 587; but I can see the maybe relevant line at 2:04:58 PM.648 ( 49 FPS), ln 2470 
```
2:04:58 PM.648 ( 49 FPS)	Exception running asynchronous task:
System.AggregateException: One or more errors occurred. ---> System.Exception: Unable to fetch search results. Current count: 0. Desired count: 60. Attempts: 5. Elapsed: 00:00:04.4431898. LastResult: CloudResult<SkyFrost.Base.SearchResults`1[FrooxEngine.Store.Record]> - State: BadRequest, Attempts: 1, Content: Cannot perform a private search when not signed in, Entity: 
  at SkyFrost.Base.RecordSearch`1+<EnsureResults>d__13[R].MoveNext () [0x001dd] in <d60eefd1cc774d57ab5e457a4cf06e6d>:0 
```


Double checking my work flow, this is actually probably a different issue where ever clicking some options (my worlds especially) breaks the world searcher until you restart on an unregistered account

**Comment by TisFoolish** [comment]
> Yeah, you can apply the component to an empty slot and then save the corresponding facet to your inventory, @TisFoolish.

I tried that but it's still broken for me, even after restarting Res. I'll get a fresh, modless log tomorrow
